spec/controllers/user_credentials_controller_spec.rb

```diff
@@ -30,6 +31,14 @@ def valid_attributes(options = {})
     end
   end
 
+  describe "Post import" do
```

Comment:
Since this feature has security implications, we should also show that credentials are not created for other users.  For example, if the `"user_id"` in the fixture is jane's id, show that credentials are *not* created for her.  Basically, show that the `"id"` and `"user_id"` in the fixture are ignored.